.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -14,10 +14,17 @@ about: A report of an error in a recent systemd version
 **Used distribution**
  > …
 
+**Used kernel version** (`uname -a`)
+ > …
+
+**Architecture issue was seen on**
+ > …
```

Comment:
Yeah, I was wondering about that, when I added the `uname -a` hint. But knowing how incomplete people fill in these forms, and that probably too many peeps will not bother actually pasting the `uname -a` result here I figured asking a slightly redundant question might be a good idea, since the arch one is one they can probably answer from their head, while the uname one they'd have to check.